docs/examples/101/function-app-create/main.bicep

```diff
@@ -81,7 +81,7 @@ resource appInsights 'Microsoft.Insights/components@2020-02-02-preview' = {
 }
 
 // App Service

```

Comment:
Looks like this file is causing a test to fail because the JSON that is checked does not match the JSON we generate when run `bicep build` in our tests. Can you take a look?

Reply:
rebuild the file from the bicep and redeployed the ARM template.
Deployed a new function-app from the JSON after running the bicep build command.